hugo/content/kommunen/karlsruhe.md

```diff
@@ -24,6 +24,15 @@ Außerdem ist eingezeichnet, wie sich die Emissionen in den nächsten Jahren ent
 
 Tipp: Ein Doppelklick auf einzelne Elemente der Legende blendet alle anderen in der Grafik aus.
 
+### Berechnung der Paris-Limits
+
+Zur Berechnung der Paris-Limits in der Grafik "Realität und Ziele" wurden folgende Daten verwendet:
```

Comment:
Falls Du magst, kannst Du hier folgenden Link einfügen (auf der Seite ist die Berechnung erklärt [falls nicht gut erklärt, gerne PR!]: `[hier wird die Berechnung erklärt](../../paris-limits)`

Übrigens gilt die Berechnung auch für den you-draw-it-chart, nicht nur für die "Realität und Ziele" Grafik.

Reply:
Erledigt.